### DISCUSSION TITLE
How to transfer a model to a website?

### DISCUSSION THREAD

**Discussion by Gleb-GG** [pyvista/pyvista#2171] [Q&A]
Hi everyone! Pyvista team, thank you for your tool, I'm incredibly pleased with what you have done.

Can you tell me, how you import the built model to the website. You have made several interactive models in your documentation. Can you tell me what your approach was? How the model was saved. Why interactive slices on the grid are presented as gif images, and the rotation of models on your site is interactive. If I run the model on a separate server, i will be able to make interactive slices of the data cube?

**Comment by akaszynski** [comment]
Hello @Gleb-GG,

We made interactive figures by leveraging the [jupyter-execute](https://jupyter-sphinx.readthedocs.io/#:~:text=Jupyter%2Dsphinx%20is%20a%20Sphinx,code%20execution%20with%20minimal%20effort.) and [pythreejs](https://github.com/jupyter-widgets/pythreejs). See our documentation regarding this in [Using pythreejs with PyVista](https://docs.pyvista.org/user-guide/jupyter/pythreejs.html#).

Not every VTK mesh type is supported at the moment, but we're working to add this based on user requests.

**Reply by Gleb-GG** [reply]
What about interactive slices, will the web be able to withstand them?
